test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -493,14 +485,309 @@ describe('v1.0.0 to v2.0.0', function () {
                             schema: 'https://schema.getpostman.com/json/collection/v2.0.0/collection.json'
                         },
                         item: [{
-                            auth: null,
                             item: []
                         }]
                     });
                     done();
                 });
             });
         });
+
+        describe('with missing properties', function () {
+            var options = {
+                inputVersion: '1.0.0',
+                outputVersion: '2.0.0',
+                retainIds: true
+            };
+
+            it('should fall back to legacy properties if auth is missing', function (done) {
+                var source = {
+                    id: '27ad5d23-f158-41e2-900d-4f81e62c0a1c',
+                    currentHelper: 'basicAuth',
+                    helperAttributes: {
+                        id: 'basic',
+                        username: 'postman',
+                        password: 'secret'
+                    }
+                };
+
+                transformer.convertSingle(source, options, function (err, result) {
+                    expect(err).to.not.be.ok;
+
+                    expect(JSON.parse(JSON.stringify(result))).to.eql({
+                        _postman_id: '27ad5d23-f158-41e2-900d-4f81e62c0a1c',
+                        name: '',
+                        request: {
+                            header: [],
+                            body: { mode: 'raw', raw: '' },
+                            auth: {
+                                type: 'basic',
+                                basic: { username: 'postman', password: 'secret', showPassword: false }
+                            }
+                        },
+                        response: []
+                    });
+                    done();
+                });
+            });
+
+            it('should discard auth creation if both: legacy and new attributes are missing', function (done) {
```

Comment:
Need to add a test case for `prioritizeV2:false` and `auth` present and `legacy` properties missing